provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -830,6 +831,30 @@ func generateSchema() schema.PackageSpec {
 					"version": {
 						TypeSpec: schema.TypeSpec{Type: "string"},
 					},
+					"kubeletExtraArgs": {
```

Comment:
I'm remembering now that TF has a ConflictsWith helper which we don't (yet?) have, could be a useful shorthand for marking mutually exclusive fields.